src/Simplify_Sub.cpp

```diff
@@ -184,7 +184,7 @@ Expr Simplify::visit(const Sub *op, ExprInfo *bounds) {
              rewrite((slice(x, c0, c1, c2) - z) - slice(y, c0, c1, c2), slice(x - y, c0, c1, c2) - z, c2 > 1 && lanes_of(x) == lanes_of(y)) ||
              rewrite((z - slice(x, c0, c1, c2)) - slice(y, c0, c1, c2), z - slice(x + y, c0, c1, c2), c2 > 1 && lanes_of(x) == lanes_of(y)) ||
 
-             (no_overflow(op->type) &&
+             (no_overflow(op->type) && EVAL_IN_LAMBDA
```

Comment:
Since this is only happening in MSVC, it might be worth only doing this `#ifdef _MSC_VER`?

Reply:
It also speeds up compile times for gcc slightly